src/codecatalyst/auth.ts

```diff
@@ -66,7 +66,9 @@ export class CodeCatalystAuthenticationProvider {
         })
 
         // set initial context in case event does not trigger
-        setCodeCatalystConnectedContext(this.isConnectionValid())
+        this.restore().then(() => {
```

Comment:
The existing CC connection wasn't being loaded in on startup, this restores the connection (if it exists) when the instance is created.